surface_flatness_node:
    ros__parameters:
        engineering:
            - "Normal"
            - "Concerte"
            - "Plaster"
            - "Partition"
            - "Veneer"
        option:
            - "TopPlateFlatness"
            - "GroundFlatness"
            - "SideWallFlatness"
        I_R_C: [0.0, 0.0, 1.0, -1.0, 0.0, 0.0, 0.0, -1.0, 0.0]    
        fileName: '/home/ubuntu/work/files/data/pointcloud.txt'